src/RegExpr.pas

```diff
@@ -3492,15 +3490,15 @@ function TRegExpr.regrepeat(p: PRegExprChar; AMax: integer): integer;
         ArrayIndex := FSubExprIndexes[Ord(opnd^)];
         if ArrayIndex < 0 then
           Exit;
-        sestart := startp[ArrayIndex];
-        if sestart = nil then
+        GrpStart := startp[ArrayIndex];
```

Comment:
Well it's ok to rename subexpressions to groups.
But can we do that in all places? Now you have FSubExprIndexes and GrpStart

Reply:
renamed to GrpIndexes/ GrpCount, okay.